### DISCUSSION TITLE
Error while sourcing

### DISCUSSION THREAD

**Discussion by Blitzboot** [kroshu/kuka_drivers#174] [Q&A] [answered]
If i follow the ReadMe i get the following error in the last step:
source install/setup.bash 
not found: "/home/user/ros2_ws/install/kuka_external_control_sdk/share/kuka_external_control_sdk/local_setup.bash"
not found: "/home/user/ros2_ws/install/kuka_external_control_sdk_examples/share/kuka_external_control_sdk_examples/local_setup.bash"

**Comment by Svastits** [comment]
`kuka_external_control_sdk` and `kuka_external_control_sdk_examples` are not ROS2 (catkin) packages, for which package.xml files were added according to [REP136](https://www.ros.org/reps/rep-0136.html) to support ROS2 tooling.
However the setup.bash files are not created for such packages, which means that everything is working, but the warnings you encounter are  displayed, when sourcing. (You would not be able to start executables from this package using `ros2 run`, but they are only libraries and the libraries they contain can be found by cmake)

**Reply by Blitzboot** [reply]
Thank you!
Okay so i just ignore this message or can i remove the error?

**Reply by Svastits** [reply, marked as answer]
We could not yet manage to remove the error in a satisfying way, so I would suggest to just ignore it...
